rustls/Cargo.toml

```diff
@@ -36,6 +36,11 @@ zlib-rs = { version = "0.2", optional = true }
 default = ["aws_lc_rs", "logging", "std", "tls12"]
 std = ["webpki/std", "pki-types/std", "once_cell/std"]
 logging = ["log"]
+# XXX TODO NAMING - ADD SEPARATORS
+# XXX TODO ADD THIS TO DEFAULT FEATURES
+defaultproviderenabled = []
```

Comment:
Would suggest splitting this out in a separate commit or PR, named `default-provider` with separate motivation on why this is hard to support on the targets you care about.

Reply:
This option is now gone